baton-maven-plugin/src/main/java/org/technologybrewery/baton/util/CommonUtils.java

```diff
@@ -0,0 +1,16 @@
+package org.technologybrewery.baton.util;
+import com.vdurmont.semver4j.Semver;
+
```

Comment:
A: let's add a comment about the intent - I assume here this is meant for common functions used while _performing_ migrations?

Reply:
Correct. Addressed